main.go

```diff
@@ -69,6 +70,10 @@ func addAllSchemes(scheme *runtime.Scheme) {
 	utilruntime.Must(openshiftv1.AddToScheme(scheme))
 
 	utilruntime.Must(configv1.AddToScheme(scheme))
+
+	utilruntime.Must(misfv1a1.AddToScheme(scheme))
+
+	pluginAddToScheme(scheme)
 	// +kubebuilder:scaffold:scheme
```

Comment:
Move the scaffolding comment above the plugin code and leave an empty line

Reply:
Updated